components/filter-components/MapFilters.vue

```diff
@@ -154,28 +155,52 @@
 
               <!-- Property titles -->
               <div class="flex pl-2 py-1 flex-wrap items-center mt-2
-                          cursor-pointer" style="width: 225px;" 
+                          cursor-pointer w-full" 
                   @click="togglePropertyShowing(property)">
                 <div class="title-point"></div>
-                <p style="font-weight: 500;"> {{ propertyItem.label }} </p>
+                <!--Descriptions-->
+                <div v-if="!isMobileDevice" class="tooltip">
+                <span class="label-and-icon">
+                  <p style="font-weight: 500; display: inline;"> {{ propertyItem.label }} </p>
+                  <div class="inline-flex items-center justify-center
+                              align-middle rounded-full border-r-2  
+                              border-l-2 border-gray-400
+                              w-5 h-5"><img src="/images/infoIcon.svg" class="pt-0.5 h-5 inline"> </div>
+                </span>
+                <span class="tooltiptext text-xs">
+                    <h4 class="font-semibold">{{propertyItem.label}}</h4>
+                    <div v-html="propertyItem.description"></div>
+                </span>
+                </div>
+                <!-- Mobile Devices -->
```

Comment:
I think there might be some issues on mobile. The dismiss X is quite small so I don't think it will meet accessibility standards. I would expect that on mobile, I could click anywhere outside of the tooltip to dismiss it but this doesn't work. Is it possible to add this in?